src/System.ComponentModel.Primitives/src/project.json

```diff
@@ -1,11 +1,24 @@
 {
   "frameworks": {
+    "netstandard1.7": {
+      "dependencies": {
+        "System.Collections.NonGeneric": "4.3.0-beta-24522-03",
+        "System.ComponentModel": "4.3.0-beta-24522-03",
+        "System.Diagnostics.Tools": "4.3.0-beta-24522-03",
+        "System.Resources.ResourceManager": "4.3.0-beta-24522-03",
+        "System.Runtime": "4.3.0-beta-24522-03",
+        "System.Threading": "4.0.0"
+      },
+      "imports": [
+        "dotnet5.8"
+      ]
+    },
     "netstandard1.0": {
```

Comment:
please do remove these dead configurations from here otherwise we do more package restoring for no reason.